elkserver/install-elkserver.sh

```diff
@@ -474,6 +474,14 @@ if [ $ERROR -ne 0 ]; then
 fi
 sleep 1
 
+echo "Installing Kibana SIEM detection rules (for MITRE ATT&CK mapping)"
+curl -X POST "http://localhost:5601/api/detection_engine/rules/_import?overwrite=true" -H 'kbn-xsrf: true' -F file=@./templates/redelk_siem_detection_rules.ndjson >> $LOGFILE 2>&1
+ERROR=$?
+if [ $ERROR -ne 0 ]; then
+    echoerror "Could not install Kibana SIEM detection rules (Error Code: $ERROR)."
+fi
+sleep 1
+
```

Comment:
The SIEM detection rules are used to enrich the data coming from rtops.
It uses the tags from `threat.technique.id` to assign it the correct TTP information and allows browsing it in the SIEM view.
The .siem-signals-* index is then used for the TTP dashboard.
SIEM detection rules run every 5 minutes.